config/convert.go

```diff
@@ -15,345 +15,11 @@
 package config
 
 import (
-	"fmt"
-	"net/url"
-	"reflect"
-
-	"github.com/alecthomas/units"
 	"github.com/coreos/container-linux-config-transpiler/config/types"
 	ignTypes "github.com/coreos/ignition/config/v2_0/types"
-	"github.com/coreos/ignition/config/validate"
 	"github.com/coreos/ignition/config/validate/report"
-	"github.com/vincent-petithory/dataurl"
-)
-
-const (
-	BYTES_PER_SECTOR = 512
 )
 
 func ConvertAs2_0_0(in types.Config) (ignTypes.Config, report.Report) {
```

Comment:
Should this exist if all it does is call a function with identical signature? Could we eliminate this and just have the caller call types.ConvertAs2_0_0 instead?